orchestration/templates/diff-table.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: diff-table
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: table-name
+          - name: gcs-bucket
+          - name: input-prefix
+          - name: staging-bq-project
+          - name: staging-bq-dataset
+          - name: jade-bq-project
+          - name: jade-bq-dataset
+      dag:
```

Comment:
This is a dag because when it is actually referenced/used, it needs to depend on other steps, right? Does this step not work properly in the main template where it is referenced if it isn't a dag?

Reply:
It's a dag because it will soon have follow-up calls to export subsets of the join table back to GCS